src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -25,5 +25,21 @@
 
 @RunWith(JUnit4.class)
 public final class UtilityTest {
-  
+  @Test
+  public void convertUsingGsonToJsonTest() {
+    // Object with attributes.
+    SubjectTag tag = new SubjectTag(5, "Interviews", "red");
+    String actual = Utility.convertToJsonUsingGson(tag);
+    String expected = "{\"id\":5,\"subject\":\"Interviews\",\"color\":\"red\"}";
+    Assert.assertEquals(expected, actual);
```

Comment:
nit: we have something at Google where we structure our tests by 'assemble, act, and assert'. For this test, it would look like the following:
```
SubjectTag tag = new SubjectTag(5, "Interviews", "red");
String expectedJson = "{\"id\":5,\"subject\":\"Interviews\",\"color\":\"red\"}";

String actual = Utility.convertToJsonUsingGson(tag);

Assert.assertEquals(expected, actual);
```

The first section is the 'assemble' section, where you put together variables and structures necessary for the test and for validating the test. The second section is the 'act' section, where you call the functionality you are testing. The final section is the 'assert' section, where you assert the results of your test.

Can you structure your tests in this manner? Thanks!!

Reply:
That is great to know! Done.